tests/gem5/configs/boot_kvm_fork_run.py

```diff
@@ -179,10 +179,10 @@
 
 # Set the Full System workload.
 motherboard.set_kernel_disk_workload(
-    kernel=Resource(
+    kernel=obtain_resource(
```

Comment:
Same thing here... should this be a workload?

Reply:
Same comment as before. It could be but I think having some set_workload examples is good too. I don't think it's in the scope of this patch to do that conversion.